doc/code_style/mom.cpp

```diff
@@ -59,36 +59,32 @@
  * Will be parsed by Doxygen.
  */
 struct another_stupid_type {
-	rofl_t blergh_gnampf;  //!< variables and functions in lowercase
-	                       //   with _ as word separator.
-	epictype42_t løl;      //!< no question, we use UTF-8 encoding.
-	ub0rtype røfëlkøptør;  //!< proper documentation is Doxygen parsable.
+	rofl_t blergh_gnampf;     //!< variables and functions in lowercase
+	                          //   with _ as word separator.
+	epictype42_t løl;         //!< no question, we use UTF-8 encoding.
+	ub0rtype     røfëlkøptør; //!< proper documentation is Doxygen parsable.
 };
 
 
-
 /**
  * Yet another Valve game. Go on, nothing to see here.
  */
 class HalfLife3 : public ValveGame {
-// vvv The access modifier is at the same indent level as the class definition:
+	// vvv The access modifier is at the same indent level as the class definition:
```

Comment:
Trying to find out where this comes from :-O

Reply:
this shouldn't matter much since we will likely never comment an access modifier. it's only important for this example file that the indent level is at 0.